### DISCUSSION TITLE
Welcome to waze-voicepack-links Discussions!

### DISCUSSION THREAD

**Discussion by pipeeeeees** [pipeeeeees/waze-voicepack-links#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪

**Comment by Ascuber** [comment]
Hi, can you help me make a custom voice model?

**Reply by pipeeeeees** [reply]
Sure thing. Just shoot me over the mp3 files on github or discord (@pipeeeeees) and I can have them uploaded at the highest possible quality

**Comment by we0ck2aw34tdsf** [comment]
Hi, I have the same problem as described in the Readme section "Debugging" when trying to use one of the voicepacks (also Android 13 phone).  The solution suggested in "Debugging" is:  "...go to the Waze App > Tap on the white button with three lines in the top left corner > Settings > Voice & Sound > Waze voice > Tap i on the custom voice pack that will not play > Scroll to the bottom and change Default voice to something different. "

On my phone I have to add another step for this solution to work as intended:  After changing the Default voice to something different, I additionally have to close Waze immediatley.  Without this step I still get only either the default voice or no voice at all.  (For closing Waze on my phone, I open the "recent apps screen" and swipe up on Waze so that it disappears from the list.)  After the next start of Waze it uses the voicepack I wanted it to use.

I hope this helps those who also have this problem.    :-)

**Comment by zdataz** [comment]
Hello, I'm looking for the Master Chief Voice in French. I only founded the English version. Can you help me, please?

**Reply by pipeeeeees** [reply]
I have been unable to find it. If I do, however, I will let you know

**Comment by Breddman** [comment]
Can you please make me a custom voice pack?

**Reply by pipeeeeees** [reply]
Sure, open an issue with the mp3 files and I will make it when I get some time to spare

**Comment by gmee12** [comment]
I've downloaded several of the packs on my phone and I see them listed, I have the circle on the left side highlighted for the voice I am wanting to use, but no matter what I do,  the voice isn't being used.  Tried updating , rebooting, and nothing I do seems to remedy the issue.  Any ideas?   I'm using a google pixel 6 pro with O.S.  with current updates etc and waze is up to date as well.

**Reply by pipeeeeees** [reply]
You could try the solution here: https://github.com/pipeeeeees/waze-voicepack-links#debugging along with various other remedies on r/waze, otherwise I am unaware of any fixes. Waze has had issues with voice packs on android for well over a year now even at latest versions of OS and app. It's unfortunate, we can only hope for a true fix.

**Comment by Anesred** [comment]
In the Grand Theft Auto IV voice pack, the phrases that arent original (metric, x reported ahead) are all in a very noticeably different, deeper voice. any plans on fixing this?

**Reply by Anesred** [reply]
I could also try myself to make a gta 4 pack from scratch, if that would be preferrable, I can upload the necessary files

**Comment by JalanTuaKong** [comment]
Anyone can make a voicepack based on the Half-Life VOX announcer? I have some audio files that can be used 🙂

**Reply by pipeeeeees** [reply]
open an issue and provide the audio files according to https://github.com/pipeeeeees/waze-voicepack-links?tab=readme-ov-file#have-mp3-files

**Comment by lejon0336-gif** [comment]
Hey pipeeeeees can you add Kaatje's voice

**Comment by WhoTookTheDoor** [comment]
Is there a way to tutorialise how to save the voicepacks so the community can help to archive them?

**Comment by webwebster** [comment]
I THINK i created a new voice, and my command line gave me an address. But the voice does not show up when i enter the URL and click on the Waze link

**Comment by cipraamaranth** [comment]
Posting this here too: Could someone please post the Cortana (from Halo) voice pack (MP3 files) in English, Imperial system (Miles) please?
